src/components/navigation/GlobalNavigation/GlobalNavigation.tsx

```diff
@@ -50,7 +51,7 @@ export const GlobalNavigation = (props: IGlobalNavigationProps) => {
       <Layout.Sider className="globalNavigation__sider" width={GlobalNavWidth}>
         <Flex vertical justify="space-between" style={{ height: '100%' }}>
           <div>
-            <SuiteLogo {...props.logo} />
+            {!props.hideSuiteLogo && <SuiteLogo {...props.logo} />}
```

Comment:
nit: personal preference, but maybe something like that with true by default? also aligned with minimap show pattern
```suggestion
            {props.showSuiteLogo && <SuiteLogo {...props.logo} />}
```

Reply:
yes good call out! often i find it simpler to read `if true` statements rather than `if !true`